backend/scripts/restart-bare-metal.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Probably should update the README.md to use this script to make things even more simple.